src/app/_components/DistrictSearch/actions.js

```diff
@@ -0,0 +1,11 @@
+'use server';
+
+import { prisma } from '@/lib/db';
+
+export const getDistrictsList = async () => {
+  const districtsList = await prisma.district.findMany({
+    orderBy: { name: 'desc' },
+  });
+  const defaultOption = { id: 'all-districts', name: 'Усі' };
+  return [defaultOption, ...districtsList];
```

Comment:
Is this `defaultOption` just a frontend abstraction? If so, I'd recommend to keep it just on the frontend so that it's "loosely-coupled" (best-practice from architectural perspective).

Reply:
Thanks! Yeah, it could be considered as a frontend abstraction and is just an option selected by default when user visits the main page.